MediaRecorder.bs

```diff
@@ -241,6 +249,25 @@ interface MediaRecorder : EventTarget {
     the value might be surpassed, not achieved, or only be achieved over a long
     period of time.</li>
 
+    <li>Let |videoKeyFrameIntervalDuration| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalDuration]]=] slot if defined. Additionally let
+    |videoKeyFrameIntervalCount| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalCount]]=] slot if defined.
+    If neither of the slots are defined, the User Agent may emit key frames as
+    it deems fit.
```

Comment:
Mostly editorial, but this describes runtime behavior as a step in a sequential algorithm, which can be confusing to a reader (e.g. "the User Agent emit key frames" should not be done before the next step here.

MediaCapture recorder already does this in steps above, so this is somewhat of an existing problem, but it would be good to follow the pattern in the step above this one where it uses this form:
> Let audioBitrate be the value of recorder’s [audioBitsPerSecond](https://pr-preview.s3.amazonaws.com/handellm/mediacapture-record/pull/216.html#dom-mediarecorder-audiobitspersecond) attribute, and constrain the configuration of recorder to [do x]

Reply:
I tried to follow yours and Youenn's comments in 0df3dba, ptal!